source/blog/2017-06-22-introducing-buildah.html.md

```diff
@@ -0,0 +1,258 @@
+
+```
+# dnf -y install make golang bats btrfs-progs-devel \
+  device-mapper-devel gpgme-devel libassuan-devel \
+  git bzip2 go-md2man runc skopeo-containers
+```
+
+After that completed I then did the following, as documented in the README.md, to build Buildah itself:
+
+```
+# mkdir ~/buildah
+# cd ~/buildah
+# export GOPATH=`pwd`
+# git clone https://github.com/projectatomic/buildah \
+ ./src/github.com/projectatomic/buildah
+# cd ./src/github.com/projectatomic/buildah
+# make
+# make install
+```
+
+To verify the install:
+
+```
+# buildah --help
+```
+
+After installing Buildah via either method  you should be able to run `man buildah` and see all of the man pages associated with the tool.  From here you can get detailed examples of each of the commands.
+
+## Let's Play!
+
+Now for the fun stuff.  I wanted to set up a little container running Fedora from the docker registry.  I simply issue the following command in BASH which sets my variable named `container` to the container's name for use later on :
+
+```
+# container=$(buildah from fedora)
+Getting image source signatures
+Copying blob  sha256:691bc14ee27487db536172a1fcdbbf956f460d1e1e1b201828e3a2bab81c5ec8
+ 72.22 MiB / 72.22 MiB   [=======================================================]
+Copying config sha256:15895ef0b3b2b4e61bf03d38f82b42011ff7f226c681705a4022ae3d1d643888
+ 0 B / 2.29 KiB [--------------------------------------------------------------]
+Writing manifest to image destination
+Storing signatures
+```
+
+This command completed relatively quickly for me (26 seconds), and most of that time was used for the initial download of the Fedora base image.
+
+Now I can verify that the image has been pulled and the container has been setup by using the buildah images and buildah containers commands respectively:
+
+```
+# buildah images
+IMAGE ID     IMAGE NAME
+dc3a60619440 docker.io/library/fedora:latest
+# buildah containers
+CONTAINER ID IMAGE ID     IMAGE NAME CONTAINER NAME
+ae93de45778a dc3a60619440 fedora     fedora-working-container
+```
+## Look Ma!  No container runtime is running!
+
+We've now downloaded an image we can manipulate, without using the Docker daemon or even having it installed on the system:
+
+```
+# systemctl status docker
+Unit docker.service could not be found.
+# rpm -qa docker
+#
+```
+
+We don't need any other container runtime, either.  This is very useful because it makes Buildah easily embeddable in automation scripts and inside other container-building tools, since neither a server process nor root access are required.
```

Comment:
Change paragraph to:
We don't need any other container runtime either.  By not installing Docker or another container runtime we save file system space, lessen memory usage and cpu processing on the system.  The daemons associated with those runtimes are not only not running, they're not even present.  As we'll seen in a future blog, Buildah also allows you to build smaller containers than Docker does.   In addition to these advantages, Buildah is easily embeddable in automation scripts and inside other container-building tools, since neither a server process nor root access are required.